client/src/components/Footer.stories.jsx

```diff
@@ -0,0 +1,17 @@
+import Footer from './Footer';
+import { MemoryRouter as Router } from 'react-router-dom';
+
+
+export default {
+  title: 'components/Footer',
+  component: Footer,
+//   argTypes: { onClick: 'onClick' },
+};
+
+const Template = (args) => (
+<Router>
```

Comment:
Instead of importing `MemoryRouter` in every stories file when needed you could also import it in your storybook `preview.js` as context for all stories files.  See the [storybook docs for Global Decorators](https://storybook.js.org/docs/react/writing-stories/decorators#global-decorators) in general and you can add the `MemoryRouter` in your `preview.js` like this:

```
import { MemoryRouter } from 'react-router-dom'
```

```
export const decorators = [
  Story => (
    <MemoryRouter>
      <GlobalStyles />
      <Story />
    </MemoryRouter>
  ),
]
```

Reply:
Thank you! Learned something new.